src/ImageSharp/Formats/Tiff/PhotometricInterpretation/CieLabTiffColor{TPixel}.cs

```diff
@@ -31,8 +31,8 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
             for (int x = 0; x < pixelRow.Length; x++)
             {
                 float l = (data[offset] & 0xFF) * 100f * Inv255;
-                var lab = new CieLab(l, (sbyte)data[offset + 1], (sbyte)data[offset + 2]);
-                var rgb = ColorSpaceConverter.ToRgb(lab);
+                CieLab lab = new(l, (sbyte)data[offset + 1], (sbyte)data[offset + 2]);
```

Comment:
I'm curious to how the data is encoded here. 

The CieLab  `A`, `B` values usually range from −128 to 127 when working with integer values.  
Casting to `sbyte` and using overflow is a neat trick but you have to be careful.

Reply:
To cite the spec:

```
Tiff is defined such that each quantity be encoded with 8 bits. This provides 256 levels of L* lightness;
256 levels (+/- 127) of a*, and 256 levels (+/- 127) of b*.  
...
Limiting the theoretically unbounded a* and b*
ranges to +/- 127 allows encoding in 8 bits without eliminating any but the most
saturated self-luminous colors. It is anticipated that the rare specialized applications
requiring support of these extreme cases would be unlikely to use CIELAB
anyway. All object colors, in fact all colors within the theoretical MacAdam limits,
fall within the +/- 127 a*/b* range.
```